sdk/python/feast/permissions/server/arrow.py

```diff
@@ -49,7 +48,9 @@ def __init__(self, access_token: str, *args, **kwargs):
 
     def call_completed(self, exception):
         if exception:
-            print(f"{AuthorizationMiddleware.__name__} received {exception}")
+            logger.error(
+                f"{AuthorizationMiddleware.__name__} encountered an exception: {exception}"
```

Comment:
The better way to log exception is `logging.exception` I m not sure what it logs if we do the `{exception}`.

Reply:
set as logging.exception. These will arrow flight sever related internal exceptions.